cluster/distvalidator.go

```diff
@@ -58,3 +64,13 @@ func (v DistValidator) HashTreeRootWith(hh *ssz.Hasher) error {
 
 	return nil
 }
+
+// PublicKey returns the validator group public key.
+func (v DistValidator) PublicKey() (*bls_sig.PublicKey, error) {
+	return tblsconv.KeyFromCore(core.PubKey(v.PubKey))
+}
+
+// PublicShare returns a peer's threshold BLS public share.
+func (v DistValidator) PublicShare(peerIdx int) (*bls_sig.PublicKey, error) {
```

Comment:
If we have this method perhaps we don't need PubShares to be public

Reply:
it is a DTO/value so fields are public